views/lounge.pug

```diff
@@ -0,0 +1,57 @@
+doctype html
+html
+    head
+        title Lounge | Werewolf Online - Tiger Team
+        link(rel="stylesheet", href="/stylesheets/lounge.css")
+        link(rel="stylesheet",href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css")
+        script(src="/javascripts/lounge.js")
+        script(src="https://kit.fontawesome.com/3f47eb379d.js")
+
+        script(src="https://code.jquery.com/jquery-3.3.1.slim.min.js")
```

Comment:
Why you import Jquery library here? I see you install Jquery library like a dependency in package.json ^^